CWE/events/Italy ahistorical.txt

```diff
@@ -3883,16 +3973,10 @@ country_event = {
 	
 	mean_time_to_happen = { days = 1 }
 	
-	immediate = { 
-	    government = democracy
-		
-        political_reform = elitist_service
-        political_reform = parties_promote_democratic
-        political_reform = lgbt_equal_rights
-        political_reform = proportional_representation
-        political_reform = universal_voting
-		political_reform = capital_punishment
-		
+	immediate = { 	
+	    remove_country_modifier = dirigisme
```

Comment:
Please do not to put modifiers in the immediate scope. It will make the game generate out the modifier’s effects in the text box which crowds out the other localisation text.